main.py

```diff
@@ -1,10 +1,48 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as json_file:
+        players = json.load(json_file)
```

Comment:
You are overloading the context manager by performing operations inside it. According to the checklist, you should exit the context manager block after reading the data. Consider moving the operations on 'players' outside the 'with' block.